beta-prompts/beta_prompts/generation_system.py

```
@@ -0,0 +1,955 @@
+            task_type="technical",
+            domain="software_engineering",
+            complexity="complex",
+            elements=[
+                "role",
+                "technical_context",
+                "requirements",
+                "thinking",
+                "output_format",
+            ],
+            template="""<role>
+You are a senior {role_type} with {experience} years of experience in {technology_stack} and expertise in {specialization}.
```

Ruff (0.12.2)

Line too long (123 > 100)

(E501)